<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>807</width>
    <height>677</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_20">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_19">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_12">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt; font-weight:600;&quot;&gt;Appointment Details&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;Patient Details&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_11">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_18">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_10">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_7">
               <item>
                <widget class="QLabel" name="label_3">
                 <property name="text">
                  <string>Patient ID:</string>
                 </property>
                 <property name="buddy">
                  <cstring>txtPatientId</cstring>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="txtPatientId">
                 <property name="readOnly">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_8">
               <item>
                <widget class="QLabel" name="label_4">
                 <property name="text">
                  <string>Name:</string>
                 </property>
                 <property name="buddy">
                  <cstring>txtPatientName</cstring>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="txtPatientName">
                 <property name="readOnly">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_9">
               <item>
                <widget class="QLabel" name="label_5">
                 <property name="text">
                  <string>Surname:</string>
                 </property>
                 <property name="buddy">
                  <cstring>txtPatientSurname</cstring>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="txtPatientSurname">
                 <property name="readOnly">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_10">
               <item>
                <widget class="QLabel" name="label_6">
                 <property name="text">
                  <string>Sex:</string>
                 </property>
                 <property name="buddy">
                  <cstring>txtPatientSex</cstring>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QLineEdit" name="txtPatientSex">
                 <property name="readOnly">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_11">
               <item>
                <widget class="QLabel" name="label_7">
                 <property name="text">
                  <string>Date of Birth:</string>
                 </property>
                 <property name="buddy">
                  <cstring>txtPatientDob</cstring>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="txtPatientDob">
                 <property name="readOnly">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_12">
               <item>
                <widget class="QLabel" name="label_13">
                 <property name="text">
                  <string>Assessment:</string>
                 </property>
                 <property name="buddy">
                  <cstring>txtPatientAssessment</cstring>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QTextBrowser" name="txtPatientAssessment"/>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_9">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_17">
               <item>
                <widget class="QLabel" name="label_8">
                 <property name="text">
                  <string>Phone:</string>
                 </property>
                 <property name="buddy">
                  <cstring>txtPatientPhone</cstring>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="txtPatientPhone">
                 <property name="readOnly">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_16">
               <item>
                <widget class="QLabel" name="label_9">
                 <property name="text">
                  <string>Height:</string>
                 </property>
                 <property name="buddy">
                  <cstring>txtPatientHeight</cstring>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="txtPatientHeight">
                 <property name="autoFillBackground">
                  <bool>false</bool>
                 </property>
                 <property name="readOnly">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_15">
               <item>
                <widget class="QLabel" name="label_10">
                 <property name="text">
                  <string>Weight:</string>
                 </property>
                 <property name="buddy">
                  <cstring>txtPatientWeight</cstring>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="txtPatientWeight">
                 <property name="readOnly">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_14">
               <item>
                <widget class="QLabel" name="label_11">
                 <property name="text">
                  <string>BMI:</string>
                 </property>
                 <property name="buddy">
                  <cstring>txtPatientBmi</cstring>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="txtPatientBmi">
                 <property name="readOnly">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_13">
               <item>
                <widget class="QLabel" name="label_12">
                 <property name="text">
                  <string>Epsworth Score:</string>
                 </property>
                 <property name="buddy">
                  <cstring>txtPatientEpsworth</cstring>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="txtPatientEpsworth">
                 <property name="readOnly">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="lineWidth">
          <number>2</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <widget class="QLabel" name="label_14">
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;Appointment&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_6">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <item>
                <widget class="QLabel" name="label_15">
                 <property name="text">
                  <string>Appointment Date:</string>
                 </property>
                 <property name="buddy">
                  <cstring>txtAppointmentDate</cstring>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="txtAppointmentDate">
                 <property name="readOnly">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout">
               <item>
                <widget class="QLabel" name="label_16">
                 <property name="text">
                  <string>Registration Date:</string>
                 </property>
                 <property name="buddy">
                  <cstring>txtAppointmentRegDate</cstring>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="txtAppointmentRegDate">
                 <property name="readOnly">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_2">
               <item>
                <widget class="QLabel" name="label_17">
                 <property name="text">
                  <string>Diagnosis:</string>
                 </property>
                 <property name="buddy">
                  <cstring>txtAppointmentDiagnosis</cstring>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPlainTextEdit" name="txtAppointmentDiagnosis"/>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout">
               <item>
                <widget class="QLabel" name="label_19">
                 <property name="text">
                  <string>Notes:</string>
                 </property>
                 <property name="buddy">
                  <cstring>txtAppointmentNotes</cstring>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPlainTextEdit" name="txtAppointmentNotes"/>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <item>
                <widget class="QLabel" name="label_18">
                 <property name="text">
                  <string>AHI:</string>
                 </property>
                 <property name="buddy">
                  <cstring>txtAppointmentAhi</cstring>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="txtAppointmentAhi"/>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QLabel" name="label_20">
               <property name="text">
                <string>Treatment:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPlainTextEdit" name="txtAppointmentTreatment"/>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_4">
               <item>
                <widget class="QLabel" name="lblPsgReport">
                 <property name="text">
                  <string>PSG Report</string>
                 </property>
                </widget>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_4">
                 <item>
                  <widget class="QPushButton" name="buttonAttachPsgReport">
                   <property name="text">
                    <string>Attach Report</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="buttonOpenPsgReport">
                   <property name="text">
                    <string>Show Report</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_3">
               <item>
                <widget class="QLabel" name="lblDocReport">
                 <property name="text">
                  <string>Doctor's Report</string>
                 </property>
                </widget>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_5">
                 <item>
                  <widget class="QPushButton" name="buttonAttachDocReport">
                   <property name="text">
                    <string>Attach Report</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="buttonOpenDocReport">
                   <property name="text">
                    <string>Show Report</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="buttonEditPatient">
         <property name="text">
          <string>Edit Patient Details</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonViewPreviousAppointments">
         <property name="text">
          <string>View Previous Appointments</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="buttonSave">
         <property name="text">
          <string>Save Appointment Changes</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonCancel">
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonCancel</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>649</x>
     <y>420</y>
    </hint>
    <hint type="destinationlabel">
     <x>649</x>
     <y>457</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
